src/commands/ext-uninstall.ts

```diff
@@ -15,6 +18,26 @@ import { requirePermissions } from "../requirePermissions";
 import * as utils from "../utils";
 import * as logger from "../logger";
 
+marked.setOptions({
+  renderer: new TerminalRenderer(),
+});
+
+/**
+ * We do not currently support uninstalling extensions that require additional uninstall steps to be taken in the CLI. Direct them to the Console to uninstall the extension.
+ * - pubsub-stream-bigquery: requires calling a public HTTP uninstall function
```

Comment:
Please delete this comment line about "pubsub-stream-bigquery", don't need to go into implementation details here.